app/javascript/redux/greetings/greetingsSlice.js

```diff
@@ -0,0 +1,24 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const API_URL = 'http://localhost:3000/api/v1/greetings';
+
+const FETCH = 'hello-rails-react/greetings/FETCH';
+
+const initialState = {
+  greeting: '',
+};
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case `${FETCH}/fulfilled`:
+      return { ...state, greeting: action.payload };
+    default:
+      return state;
+  }
+};
+
+export const fetchGreeting = createAsyncThunk(FETCH, async () => {
```

Comment:
Well done 👍🏼 , the action to fetch greetings from backend is added

Reply:
Thank you sincerely for the valuable feedback and kind appreciation; it has significantly elevated my motivation to new heights. 😊😊😊